src/partition/repart.c

```diff
@@ -7040,7 +7041,7 @@ static int parse_argv(int argc, char *argv[]) {
                         return log_oom();
         }
 
-        arg_node = argc > optind ? argv[optind] : NULL;
+        arg_node = argc > optind ? strdup(argv[optind]) : NULL;
```

Comment:
OOM check?

Please write

```c
if (argc > optind) {
        arg_node = strdup(...);
        if (!arg_node)
                ...
}
```

There's no need to assign `NULL` anyway.

Reply:
Done, thanks.